config.json

```diff
@@ -366,7 +366,7 @@
         "name": "Leap",
         "uuid": "7c8294ee-5924-4bf8-a72f-31d0e2d7d9a0",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["numbers"],
```

Comment:
type-conversion

Reply:
Why ? What is your reasoning ?

Reply:
The `new Date(...)`  solution, which, technically, is more *correct* than the one using modulo :P

Reply:
But maybe we don't add it as a _prereq_.